### DISCUSSION TITLE
Страйк за видео на ютубе

### DISCUSSION THREAD

**Discussion by GoSamOfficial** [bol-van/zapret#842] [Q&A]
Добрый день. Я на почту вам написал но ответа не дождался. Честно говоря не могу поверить что страйк сделан вами добровольно. Может ркн до вас добрался и попросил все видео обозревающие программу удалить. Но будучи владельцем канала не могу не попробовать попросить отозвать страйк

**Comment by GoSamOfficial** [comment]
У меня есть предположение что страйк липовый от РКН, потому что в заявлении даже нет имени отправителя или владельца контента

**Comment by bol-van** [comment]
У РКН есть другие способы. Он уже внес в список адрес этого репозитория.
Гитхаб они еще пока не решились забанить весь, но яндекс уже перестал выдавать в поиске.
Вряд ли РКН будет охотиться за каждым видео на заблокированном youtube.
Почту свою не разглашаю, ничего не получал. 
Если нужно пообщаться в личке, пишите в личку на ntc.party.

**Comment by GoSamOfficial** [comment]
> У РКН есть другие способы. Он уже внес в список адрес этого репозитория. Гитхаб они еще пока не решились забанить весь, но яндекс уже перестал выдавать в поиске. Вряд ли РКН будет охотиться за каждым видео на заблокированном youtube. Почту свою не разглашаю, ничего не получал. Если нужно пообщаться в личке, пишите в личку на ntc.party.

у меня там новый аккаунт, напишу как только разблокируют лс. Вы являетесь создателем программы zapret? И даёте ли вы разрешение на съмку видео с гайдом как пользоваться программой?

Видео с ютуба удалили, но у меня остался оригинал, могу прислать если нужно. Я не хочу его восстанавливать но возможно с вашей помощью сниму страйк

**Comment by bol-van** [comment]
Я и не даю, и не запрещаю. Вообще не хочу в это вмешиваться.
Но могу сказать, что не запрещал

**Comment by GoSamOfficial** [comment]
> Я и не даю, и не запрещаю. Вообще не хочу в это вмешиваться. Но могу сказать, что не запрещал

![image](https://github.com/user-attachments/assets/da4c18c2-5e7f-4bdf-a924-3e232faab0ea)

**Comment by GoSamOfficial** [comment]
Могу в личку или куда угодно все свои ролики прислать. Я прямо на видео скачивал с официальных гитхабов. Так же в одном ролике говорил про возможные мошенничества и призывал скачивать только с официальных страниц.

**Comment by bol-van** [comment]
Я не судья и не цензор. Мне вообще ничего не известно о чьих-то левых требованиях. Не имею к ним никакого отношения.

Можете дать ссылку на этот пост администрации ресурса.

I have never asked anyone to remove any links to my software.
I dont care what they do with it, how they use it if no harm is involved.
Even if it is I can do little to prevent it from happening.

This software is free and open source under MIT license as stated in docs/LICENSE.txt
Anyone can use it, modify or redistribute. And none can require others to delete it.
If there're viruses or any other harm - it's the special case and I agree it would be good to delete such links and videos. But if not - I see no problem

**Comment by GoSamOfficial** [comment]
> Я не судья и не цензор. Мне вообще ничего не известно о чьих-то левых требованиях. Не имею к ним никакого отношения.
> 
> Можете дать ссылку на этот пост администрации ресурса.
> 
> I have never asked anyone to remove any links to my software. I dont care what they do with it, how they use it if no harm is involved. Even if it is I can do little to prevent it from happening.
> 
> This software is free and open source under MIT license as stated in docs/LICENSE.txt Anyone can use it, modify or redistribute. And none can require others to delete it. If there're viruses or any other harm - it's the special case and I agree it would be good to delete such links and videos. But if not - I see no problem

спасибо. Вам известно что-нибудь про этот репозиторий https://github.com/Flowseal/zapret-discord-youtube?

**Comment by bol-van** [comment]
Это чья-то сборка с предопределенными в батниках стратегиями по принципу попробуй 1 2 3 . Авось что-нибудь сработает.
Судя по обсуждениям на форумах, к вирусам она не относится

**Comment by GoSamOfficial** [comment]
> Это чья-то сборка с предопределенными в батниках стратегиями по принципу попробуй 1 2 3 . Авось что-нибудь сработает. Судя по обсуждениям на форумах, к вирусам она не относится

вот про этот репозиторий я рассказывал, и моё видео копистрайнули и удалили. Вы точно не знаете этого Никиту который мне написал на почту?

**Comment by bol-van** [comment]
Точно не знаю

**Comment by numbereleven-a** [comment]
> вот про этот репозиторий я рассказывал, и моё видео копистрайнули и удалили. 

так может вам тогда туда? https://github.com/Flowseal/zapret-discord-youtube/issues
ps
хотя мне кажется что страйк кидают те кто вирусную распостроняют

**Comment by Fenrir-0xFF** [comment]
> Почту свою не разглашаю, ничего не получал.
> Если нужно пообщаться в личке, пишите в личку на ntc.party. 

Кстати было бы не плохо выложить какойнибуть способ для быстрой анонимной связи без регистрации, канал в IRC или Tox ID 
Tox надёжнее но IRC канал предпочтительнее, чтобы народ постоянно не выносил мозги в личку, а в общем чяте другие люди могут подсказать как настроить тем у кого не работает. 
  
Насчёт форума - там движок очень ущербный, все страницы на java скриптах основаны которые ещё и глючят, в чём проблема поставить стиль поумолчянию без слишком мудрёных js или какойнибуть phpBB движок ХЗ.

**Reply by ImMALWARE** [reply]
tox, irc... какой ужас... Matrix давно изобрели.

**Comment by numbereleven-a** [comment]
> Кстати было бы не плохо выложить какойнибуть способ для быстрой анонимной связи без регистрации, канал в IRC или Tox ID

99% сообщений будет в стиле "у меня не работает запрет что делать помогите"

**Comment by Fenrir-0xFF** [comment]
> 99% сообщений будет в стиле "у меня не работает запрет что делать помогите"

На то и расчёт по большей чясти, зато гитхаб разгрузят темболее репозиторий для рашки могут закрыть в любой момент, да и с регистрацией возиться не придётся людям. 
На гитхабе в идеале выложить на видное место какоенибуть сообщение которое будет обновляться или както issue закрепить наверх в котором будут актуальные настройки для разных провайдеров

**Comment by numbereleven-a** [comment]
нет

**Comment by bol-van** [comment]
Цель данного репозитория - не производить таблетки, а делать движок.
Нести простоту в массы не является его задачей.
От этого автор давно открестился и отдал на реализацию всем желающим.
Вот сами и следите за провайдерами, делитесь. В дискуссии, на форумы, в IRC, куда угодно.
Здесь только движок и технически разумные его обсуждения

**Comment by Fenrir-0xFF** [comment]
>Цель данного репозитория - не производить таблетки, а делать движок....
> Здесь только движок и технически разумные его обсуждения

Об этом и речь, чтоб только по работе движка на гитхаб писали. 

>Вот сами и следите за провайдерами, делитесь. В дискуссии, на форумы, в IRC, куда угодно. 

Естественно тебя этим грузить никто не станет. 
IRC сервак в понедельник или вторник сам поищю если никто не предложит реньше, пока что на примете ilita.i2p помимо обычного интернета работает в tor и i2p (по крайней мере пол года назад так было).   

По поводу провайдеров: возникает чисто технический вопрос по возможностям гитхаба - если к примеру создал тему в дискуссии - ты можеш в эту тему назначить модераторов чтобы могли редактировать первое сообщение и удалять не актуальные коментарии чтобы не разводить помойку? У меня к примеру нет возможности следить за ней каждый день надо чтобы хотябы несколько человек было которые могут её редактировать, желающие полюбому найдутся, но есть ли техническая возможность? 

>Нести простоту в массы не является его задачей.

Вот за это тебе уважуха! 
И кучя проклятий в адрес создателя GoodbyeDPI, благодаря стараниям которого паразиты начяли лазейки прикрывать. 
Щяс правда не уверен что это уже актуально, но пускай zapret так и остаётся только для илиты) 

А так речь обовсём этом изначяльно была о том чтобы люди у которых не хватает мозгов хотябы простым перебором подобрать рабочюю стратегию под своего провайдера не разводили тут срачи. И както связаться с разработчиком с критическими проблемами без регистраций на гитхабе или форуме.

**Comment by Elllkere** [comment]
> вот про этот репозиторий я рассказывал, и моё видео копистрайнули и удалили.

Интересные обстоятельства, если это правда. Люди берут готовый инструмент и лишь подбирают болтики, а потом еще и страйкуют за какое-то АП.

> На гитхабе в идеале выложить на видное место какоенибуть сообщение которое будет обновляться или както issue закрепить наверх в котором будут актуальные настройки для разных провайдеров

Не "выпендриваюсь", но я уже сделал програмку с менюшкой, которую обновляю https://github.com/Elllkere/misterfishdpi и об обновлении пользователи узнают при запуске программы, там же и получают ссылку на скачивание. Единственное я не делал каких-то авто обновлений или что-то еще, все через гитхаб качается, но я не вижу сложного скачать рарник из открытой страницы, а на случий полной блокировки гитхаба или отдельного репозитория можно будет добавить.

**Comment by Fenrir-0xFF** [comment]
> а потом еще и страйкуют за какое-то АП.

Переходи с ютюба на https://framatube.org/ там вроде страйкать нельзя, да и NewPipe его поддерживает, темболее ютюб монитезацию отключил. 


> но я уже сделал програмку с менюшкой, которую обновляю

Она же только под винду у тебя. Кстати почему rar а не zip? zip более универсален, на крайняк 7z

**Comment by electrifying** [comment]
> но пускай zapret так и остаётся только для илиты)

Да для какой элиты вы о чем, посмотрите сколько звезд у репозитория с форком запрета только однокнопочным с готовыми батниками, если ркн захочет запрет отлетит со временем так же как и goodbyedpi. Если бы автор никаких сборок под винду не делал, а оставил чисто линукс решением тогда да можно было бы считать элитным, хотя гайдов по установке на роутер написано не мало, но это все равно сложней чем нажать два раза на батник для многих

**Comment by Elllkere** [comment]
> Она же только под винду у тебя

Ну да, если ты понимаешь как поставить и пользоваться линуксом, то я сомневаюсь что тебе нужна прога , где 1 кнопкой включается. Это в первую очередь сделано для обычных людей, которые просто пользуются компом и хотят без проблем заходить на сайты.

> Кстати почему rar а не zip?

Просто привык, winrar -> сделать архив -> .rar

**Comment by GoSamOfficial** [comment]
@bol-van 
![image](https://github.com/user-attachments/assets/96721915-159a-4db8-8f8f-8519d0b3208c)
Можно я с вашего разрешения подам встречную жалобу? Там достаточно письменного разрешения на использование контента

**Comment by GoSamOfficial** [comment]
хоть и очевидно что в лицензии всё расписано https://github.com/bol-van/zapret/blob/master/docs/LICENSE.txt
